tests/test_actions.py

```diff
@@ -168,10 +180,10 @@ def test_valueerror_when_poll_is_larger_than_timeout(self, Tester):
 
         assert "poll must be less than or equal to timeout" in str(actual_exception)
 
-    @mock.patch("screenpy.actions.eventually.time")
+    @mock.patch("screenpy.actions.eventually.time", autospec=True)
     def test_perform_eventually_times_out(self, mocked_time, Tester):
         num_calls = 5
-        mocked_time.time = mock.Mock(side_effect=[1] * num_calls + [100])
+        mocked_time.time = mock.create_autospec(time.time, side_effect=[1] * num_calls + [100])
```

Comment:
So i take it `autospec` on the `time` module does not reach down deeply enough to hit `time.time`? Can we mock `screenpy.actions.eventually.time.time`?

Maybe we could adjust how `time` is used in `Eventually` to just import the internal `time` instead, that would make this test easier.